pyproject.toml

```diff
@@ -134,7 +134,14 @@ integration = 'pytest --maxfail=5 -m "integration" test'
 integration-mac = 'pytest --maxfail=5 -m "integration" test -k "not tika"'
 integration-windows = 'pytest --maxfail=5 -m "integration" test -k "not tika"'
 types = "mypy --install-types --non-interactive --cache-dir=.mypy_cache/ {args:haystack}"
-lint = "pylint -ry -j 0 {args:haystack}"
+lint = [
```

Comment:
fixes the first warning here

<img width="1371" alt="Screenshot 2024-02-29 at 15 54 40" src="https://github.com/deepset-ai/haystack/assets/7667273/3584be49-d661-49c3-80d7-692555f95ef4">